### PR TITLE
chore: removing unused dependencies and improving stories

### PR DESCRIPTION
**Pull request by DeividVeloso**
- Removing unused dependencies and improving stories

## 📝 To Do

- [x] All linters pass
- [ ] All tests pass
- [ ] Change documentation based on my changes

## ✋ Manual testing

1. Run `npm i`
2. Run `npm run storybook`

### Desktop

- [ ] Edge (latest)
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

## 📸 Screenshots

| Description | Screenshot |
| ----------- | ---------- |
| Desktop     |            |
| usage.gif   |            |

### CODE REVIEW

File: packages/js/examples/react-audio-widgets/stories/Audio.stories.js [hunk 1]
```diff
@@ -5,20 +5,32 @@ import WebDialerComponent from './components/WebDialer';
...
+
+WebDialer.args = {
+  token: 'JWT_TOKEN',
+  callerName: 'Caller ID Name',
+  callerNumber: 'Caller ID Number',
+  disableMicrophone: false,
+  username: process.env.STORYBOOK_USERNAME,
+  password: process.env.STORYBOOK_PASSWORD,
+  defaultDestination: process.env.STORYBOOK_DESTINATION || '18004377950',
+};
```
Comment (arghav) [inline packages/js/examples/react-audio-widgets/stories/Audio.stories.js:36]: Missing new line at the end. You can set this in your IDE.

File: packages/js/examples/react-audio-widgets/stories/Audio.stories.js [hunk 2]
```diff
@@ -5,20 +5,32 @@ import WebDialerComponent from './components/WebDialer';
...
   <ClickToCallComponent {...args} {...storybook.globals.telnyxAuth} />
 );
 
+ClickToCall.args = {
+  token: 'JWT_TOKEN',
+  callerName: 'Caller ID Name',
+  callerNumber: 'Caller ID Number',
+  username: process.env.STORYBOOK_USERNAME,
+  password: process.env.STORYBOOK_PASSWORD,
+  defaultDestination: process.env.STORYBOOK_DESTINATION || '18004377950',
```
Comment (arghav) [inline packages/js/examples/react-audio-widgets/stories/Audio.stories.js:21]: Do we need the default value `'18004377950'`?

File: packages/js/examples/react-audio-widgets/stories/Audio.stories.js [hunk 3]
```diff
@@ -5,20 +5,32 @@ import WebDialerComponent from './components/WebDialer';
...
 );
+
+WebDialer.args = {
+  token: 'JWT_TOKEN',
+  callerName: 'Caller ID Name',
+  callerNumber: 'Caller ID Number',
+  disableMicrophone: false,
+  username: process.env.STORYBOOK_USERNAME,
+  password: process.env.STORYBOOK_PASSWORD,
+  defaultDestination: process.env.STORYBOOK_DESTINATION || '18004377950',
```
Comment (arghav) [inline packages/js/examples/react-audio-widgets/stories/Audio.stories.js:35]: Do we need the default value `'18004377950'`?

File: packages/js/examples/react-audio-widgets/stories/Utilities.stories.js [hunk 4]
```diff
@@ -4,15 +4,14 @@ import Utilities from './components/Utilities';
...
 };
 
 export const Examples = (args, storybook) => (
   <Utilities {...args} {...storybook.globals.telnyxAuth} />
 );
+
+Examples.args = {
+  username: process.env.STORYBOOK_USERNAME,
+  password: process.env.STORYBOOK_PASSWORD,
+}
```
Comment (arghav) [inline packages/js/examples/react-audio-widgets/stories/Utilities.stories.js:17]: Missing new line at the end. You can set this in your IDE.

**Review by arghav — CHANGES_REQUESTED** [review]

File: packages/js/examples/react-audio-widgets/stories/Audio.stories.js (hunk 3 above)
Comment (DeividVeloso) [reply packages/js/examples/react-audio-widgets/stories/Audio.stories.js:35]: yes, it is a echo number for test.

File: packages/js/examples/react-audio-widgets/stories/Audio.stories.js [hunk 5]
```diff
@@ -5,20 +5,32 @@ import WebDialerComponent from './components/WebDialer';
...
   title: 'Audio',
-  args: {
-    callerName: 'Caller ID Name',
-    callerNumber: 'Caller ID Number',
-    disableMicrophone: false,
-    username: process.env.STORYBOOK_USERNAME,
-    password: process.env.STORYBOOK_PASSWORD,
-    defaultDestination: process.env.STORYBOOK_DESTINATION || '18004377950',
-  },
+
```
Comment (muriware) [inline packages/js/examples/react-audio-widgets/stories/Audio.stories.js:8]: Is the empty line intended?

File: packages/js/examples/react-audio-widgets/stories/Utilities.stories.js [hunk 6]
```diff
@@ -4,15 +4,14 @@ import Utilities from './components/Utilities';
...
 export default {
   title: 'Utilities',
-  args: {
-    callerName: 'Caller ID Name',
-    callerNumber: 'Caller ID Number',
-    username: process.env.STORYBOOK_USERNAME,
-    password: process.env.STORYBOOK_PASSWORD,
-    defaultDestination: process.env.STORYBOOK_DESTINATION || '18004377950',
-  },
+
```
Comment (muriware) [inline packages/js/examples/react-audio-widgets/stories/Utilities.stories.js:7]: Is the empty line intended?

File: packages/js/examples/react-audio-widgets/stories/Audio.stories.js (hunk 5 above)
Comment (DeividVeloso) [reply packages/js/examples/react-audio-widgets/stories/Audio.stories.js:8]: Yes, it is.

File: packages/js/examples/react-audio-widgets/stories/Utilities.stories.js (hunk 6 above)
Comment (DeividVeloso) [reply packages/js/examples/react-audio-widgets/stories/Utilities.stories.js:7]: Yes, it is.

Comment (DeividVeloso) [reply packages/js/examples/react-audio-widgets/stories/Utilities.stories.js:7]: Sorry, it is not.

File: packages/js/examples/react-audio-widgets/stories/Audio.stories.js (hunk 5 above)
Comment (DeividVeloso) [reply packages/js/examples/react-audio-widgets/stories/Audio.stories.js]: Sorry, it is not.

Comment (muriware) [reply packages/js/examples/react-audio-widgets/stories/Audio.stories.js]: No problem. 😃

File: packages/js/examples/react-audio-widgets/stories/Utilities.stories.js (hunk 6 above)
Comment (muriware) [reply packages/js/examples/react-audio-widgets/stories/Utilities.stories.js]: No problem. 😃

**Review by muriware — APPROVED** [review]
LGTM! 🎉

**Pull request merged**